### PR TITLE
[PR] Support to export overlay view

### PR DESCRIPTION
**Pull request by tucan9389**
## Cautions

- This PR will be merged to https://github.com/tucan9389/PoseEstimation-TFLiteSwift/pull/50
- ⚠️**This PR should be merged first, and then #50 be merged.**

## Related Issue

- #52 

## PR Point

- Add the export button on navigation right bar for #52

### CODE REVIEW

File: PoseEstimation-TFLiteSwift/StillImageHeatmapViewController.swift [hunk 1]
```diff
@@ -205,6 +205,19 @@ class StillImageHeatmapViewController: UIViewController {
...
     }
+    
+    @IBAction func export(_ sender: Any) {
+        guard let topPaletteViewRect = topPaletteView?.frame,
+            let overlayViewRect = overlayHeatmapView?.frame else { return }
+        let rect = CGRect(x: topPaletteViewRect.origin.x, y: topPaletteViewRect.origin.y, width: overlayViewRect.width, height: topPaletteViewRect.height + overlayViewRect.height)
+        let image = view.uiImage(in: rect)
+        let imageData = image.jpegData(compressionQuality: 0.95)
+        let paths = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)
+        let fileURL = paths[0].appendingPathComponent("pose-heatmap-demo.jpeg")
```
Comment (MBKwon) [inline PoseEstimation-TFLiteSwift/StillImageHeatmapViewController.swift:216]: 이런건 first 등으로 optional로 안전하게 접근하는게 좋아요. 혹시라도 첫번째가 비어있으면 크래시 꽝

Comment (tucan9389) [reply PoseEstimation-TFLiteSwift/StillImageHeatmapViewController.swift:216]: Ops. I accept your suggestion. Thanks. 👍

File: PoseEstimation-TFLiteSwift/StillImageLineViewController.swift [hunk 2]
```diff
@@ -198,6 +198,17 @@ class StillImageLineViewController: UIViewController {
...
     
+    @IBAction func export(_ sender: Any) {
+        guard let overlayViewRect = overlayLineDotView?.frame else { return }
+        let image = view.uiImage(in: overlayViewRect)
+        let imageData = image.jpegData(compressionQuality: 0.95)
+        let paths = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)
+        let fileURL = paths[0].appendingPathComponent("pose-linedot-demo.jpeg")
+        try? imageData?.write(to: fileURL)
+        let vc = UIActivityViewController(activityItems: [fileURL], applicationActivities: nil)
+        present(vc, animated: true)
```
Comment (syjdev) [inline PoseEstimation-TFLiteSwift/StillImageLineViewController.swift:209]: I think, export has 2 tasks.

 1. export Image.
 2. show UIActivityViewController.

I recommend to make 2 abstract task(exportImage, showUIActivityViewController).

Comment (tucan9389) [reply PoseEstimation-TFLiteSwift/StillImageLineViewController.swift:209]: @syjdev 
Thanks for your opinion. 
I think the above function has exporting behavior in the app term. So I'll separate the `export(_:)` function to saving the image step and show `UIActivityViewController` step. If you have any other comments, reply feel free.

Thank you.

**Pull request merged**